website_geoengine_store_locator/readme/CONTIBUTORS.rst

```diff
@@ -0,0 +1,2 @@
+* Stéphane Brunner <stephane.brunner@camptocamp.com>  
```

Comment:
Some typo on the file name. (missing an R)

Reply:
Corrected